tests/slow/src/test/scala/tests/feature/ShowTastyLspSuite.scala

```diff
@@ -18,14 +19,62 @@ class ShowTastyLspSuite extends BaseLspSuite("showTasty") {
             |}
             |/app/src/main/scala/a/b/Main.scala
             |package a.b
-            |object Main {
-            |  println(5)
-            |}
+            |object Main
             |""".stripMargin
       )
       _ <- server.didOpen("app/src/main/scala/a/b/Main.scala")
       result <- server.showTasty(
-        s"$workspace/app/src/main/scala/a/b/Main.scala"
+        s"$workspace/app/src/main/scala/a/b/Main.scala",
+        new l.Position(1, 1)
+      )
+    } yield
+      if (result.isRight) ()
+      else Assertions.fail("Command shouldn't failed")
```

Comment:
What I meant here is to use:
```suggestion
      assertEquals(
        result,
        Right(
        """|actual tasty content
            |""".stripMArgin
       )
     )            
```
so that we can see that it's actually working

Reply:
Can we wait with this change until we switch from webview to virtual documents?